manifests/P/PlasmoApp/plasmo_voice/1.x/1.2.x/1.2.9.yaml

```diff
@@ -0,0 +1,100 @@
+# yaml-language-server: $schema=../../../../../../doc/validation-schemas/4.0/version-manifest.json
+
+#1.18
+- loaders:
+    - fabric
+  minecraftVersions:
+    - "1.18.1"
+    - "1.18"
+  environment:
+    server: optional
+    client: optional
+  channel: release
+  depends: ~
+  bundles: ~
+  breaks: ~
+  conflicts: ~
+  recommends: ~
+  thirdPartyIds:
+    modrinth: "1bZhdhsH"
+    curseforge: ~
+  license: ARR
+  fileType: jar
+  md5: 14549a5f4cef6ca6f781d0847fa54e44
+  downloadPageUrls:
+    modrinth: https://modrinth.com/mod/plasmo-voice/version/apgep7HE
+    curseforge: ~
+    sourceControl: https://github.com/plasmoapp/plasmo-voice/releases/tag/1.2.9
+    others: ~
+  fileUrls:
+    modrinth: https://cdn.modrinth.com/data/1bZhdhsH/versions/fabric-1.18-1.2.9/plasmovoice-fabric-1.18.1-1.2.9.jar
+    curseforge: ~
+    sourceControl: https://github.com/plasmoapp/plasmo-voice/releases/download/1.2.9/plasmovoice-fabric-1.18.1-1.2.9.jar
+    others: ~
+
+#1.17
+- loaders:
+    - fabric
+  minecraftVersions:
+    - "1.17.1"
+    - "1.17"
+  environment:
+    server: optional
+    client: optional
+  channel: release
+  depends: ~
+  bundles: ~
+  breaks: ~
+  conflicts: ~
+  recommends: ~
```

Comment:
The `fabric.mod.json` has `"soundphysics": "*"` under `suggests`, and after some thought I'd say they meant `recommends`, so you should add it to our `recommends`:
```yaml
  recommends:
    - packageId: vlad2305m.soundphysics
      version: "*"
```